design/datetimepicker-reference.html

```
@@ -0,0 +1,480 @@
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td><code>value</code></td>
+            <td><code>string?</code></td>
+            <td>—</td>
+            <td>ISO datetime string (ej: <code>2026-04-03T09:30:00</code>)</td>
+          </tr>
+          <tr>
+            <td><code>onChange</code></td>
+            <td><code>(iso: string) => void</code></td>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)